src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+    @Override
+    public List<Truck> get() {
+        List<Truck> trucks = new ArrayList<>();
+        trucks.add(new Truck(2000));
```

Comment:
I think you should creat these objects using for loop and constant field like MACHINE_COUNT